SpatialGDK/Source/SpatialGDK/Public/EngineClasses/SpatialActorChannel.h

```diff
@@ -310,6 +310,8 @@ class SPATIALGDK_API USpatialActorChannel : public UActorChannel
 
 	uint8 FramesTillDormancyAllowed = 0;
 
+	uint32 ReplicationBytesWritten = 0;
```

Comment:
Clarification that this isn't for the lifetime, but per frame

Reply:
+1